docs/Reference/Consensus/IBFT-RPC-API.md

```diff
@@ -169,3 +169,39 @@ istanbul.getSignersFromBlockByHash(blockHash)
 - `hash`: `String` - The retrieved block's hash
 - `author`: `String` - The address of the block proposer
 - `committers`: `[]String` - The list of all addresses whose seal appears in this block
+
+### istanbul.status
+
+Returns the signing status of blocks for a given block range. If start block number and end block number are not provided then it returns the status of last 64 blocks by default.
```

Comment:
Suggestion: "If the start block and end block numbers are not provided, it returns the status of the last 64 blocks by default."

Reply:
done